bower.json

```diff
@@ -24,7 +24,7 @@
     "doc-style/**"
   ],
   "dependencies": {
-    "adapterjs": "0.14.1",
+    "adapterjs": "0.14.2",
```

Comment:
@ncurrier because adapterjs end needs to honour the "mediaSource" flag in chrome, and also in the extension.